sentences/nb/_common.yaml

```diff
@@ -8,6 +8,11 @@ responses:
     no_entity: "Finner ingen enhet med navn {{ entity }}"
     handle_error: "En uventet feil oppsto"
 lists:
+  brightness:
```

Comment:
No changes to _common required. Your proposals are already merged in with #904. Note that they have changed requirements to no whitespaces around |, stated in new hassil  #830.